docs/_partner_api/changelog.md

```diff
@@ -7,6 +7,12 @@ redirect_from: /partner_api/
 
 # Changelog
 
+## 2020-01-13
+
+### Introducing rate limiting
+
+The endpoints are now rate limit protected. The limit is set to 9,000 requets per hour for an individual endpoint.
```

Comment:
`requests`

"for an individual endpoint per client_id."

Reply:
Calling out `client_id` gets a bit tricky.  Not all endpoints require a client_id.  In that case I use the access token as my key.

In the case where both are required, I default to using only the `client_id` since anyone can self generate more access tokens to skirt the limits.